editor/history_dock.cpp

```diff
@@ -219,6 +219,8 @@ void HistoryDock::_notification(int p_notification) {
 }
 
 HistoryDock::HistoryDock() {
+	set_name("History");
```

Comment:
This should have used `TTR` to be translatable I think.

Reply:
Actually no, since that's the name saved in the editor layout and we don't want to localize it (otherwise the layout breaks when changing language).

Reply:
I think node name shouldn't be translated. For dock title, we can do i18n for `TabContainer::set_tab_title` in future.